crates/rethnet_evm/src/db/layered_db.rs

```diff
@@ -250,12 +250,7 @@ impl Database for LayeredDatabase<RethnetLayer> {
         log::debug!("account with address `{}`: {:?}", address, account);
 
         // TODO: Move this out of LayeredDatabase when forking
-        Ok(account.or(Some(AccountInfo {
-            balance: U256::ZERO,
-            nonce: 0,
-            code_hash: KECCAK_EMPTY,
-            code: None,
-        })))
+        Ok(account)
```

Comment:
Feel free to remove the `TODO` comment.

You might have to move the `.or(..)` one layer up, in the `ForkDatabase`

Reply:
thanks, removed in https://github.com/NomicFoundation/hardhat/pull/3575/commits/3607c46a64fe31c608a501413cdbfa4c108951fb.

i'm not sure what you meant by moving up the `.or(..)`. is that not covered by the `else { Ok(None) }` branch in `basic()`?

Reply:
Feel free to look into this in your `napi` PR if you run into that problem there

Reply:
I'll leave this as-is for now, and if it's a problem in the broader test run (`yarn test:forking`) then I'll fix it over in [the n-api PR](https://github.com/NomicFoundation/hardhat/pull/3612)